libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/TriggerConditions/Microsoft.OnAssignEntity.schema

```diff
@@ -0,0 +1,31 @@
+{
+    "$schema": "https://raw.githubusercontent.com/microsoft/botbuilder-dotnet/master/schemas/component.schema",
+    "$role": "union(Microsoft.ITriggerCondition)",
+    "title": "Assign an entity to a property",
+    "description": "This defines the actions to take when an entity should be assigned to a property",
```

Comment:
Can you update the language around this to match the other Onxxx definitions?
    "title": "On assign entity",
    "description": "Actions to perform when an entity should be processed into a property..",